mungegithub/mungers/flake-manager.go

```diff
@@ -79,7 +79,7 @@ func (p *FlakeManager) Initialize(config *github.Config, features *features.Feat
 	p.config = config
 	p.googleGCSBucketUtils = utils.NewUtils(utils.KubekinsBucket, utils.LogDir)
 
-	var owner *testowner.ReloadingOwnerList
```

Comment:
Yup!

On Tue, Jun 14, 2016 at 5:04 PM, Ryan Hitchman <notifications@github.com>
wrote:

> In mungegithub/mungers/flake-manager.go
> <https://github.com/kubernetes/contrib/pull/1208#discussion_r67079652>:
>
> > @@ -79,7 +79,7 @@ func (p *FlakeManager) Initialize(config *github.Config, features *features.Feat
> >  	p.config = config
> >  	p.googleGCSBucketUtils = utils.NewUtils(utils.KubekinsBucket, utils.LogDir)
> >
> > -	var owner *testowner.ReloadingOwnerList
>
> Ahh, so the implicit conversion from *testowner.ReloadingOwnerList to
> sync.OwnerMapper creates a non-nil interface with type pointing at
> *testowner.ReloadingOwnerList and the data pointing at nil.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1208/files/8bc4ad3776eebe10abcec86066c4def8e29bf652..51fea8fe2e69299f1a2e660a4e1df24f0c40ee32#r67079652>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglhK6bF7o96oz29ILaJdbXF0JU0Yzks5qL0GCgaJpZM4I10u5>
> .
>